source/_components/fan.insteon_plm.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: "Insteon PLM Fan"
+description: "Instructions how to setup the Insteon PLM switches locally within Home Assistant."
```

Comment:
I'm not sure about the Insteon terminology, but maybe it should be written "fans" instead of "switches"?